lib/LedgerSMB/Upgrade_Tests.pm

```diff
@@ -137,7 +146,7 @@ Repair query column to run once per result
 
 =cut
 
-has column => (is => 'ro', isa => 'Str', required => 0);
+has column => (is => 'ro', isa => 'ArrayRef[Str]', required => 0);
```

Comment:
should we make this a plural? `column` suggests one as it's a singular. I think the rest of the code uses plurals for arrays?